Source/Config/Mixins/Templates/MoneyInput.lua

```diff
@@ -0,0 +1,24 @@
+AuctionatorConfigMoneyInputMixin = {}
+
+function AuctionatorConfigMoneyInputMixin:OnLoad()
+  self.MoneyInput.hideCopper = true
```

Comment:
Can we just remove this?

Reply:
Should be able to since it was added as a KeyValue in the XML...